src/test/_user.spec.js

```diff
@@ -132,9 +132,9 @@ describe('Test user login, signup and account verification', () => {
         });
     });
 
-    it('it should login the user into an account', done => {
+    it.skip('it should login the user into an account', done => {
```

Comment:
I guess you forgot to remove this `skip`.

Reply:
No, I did not forget to remove this, the test is actually dependent which isn't supposed to be. Hence it fails when it should not. @timi-codes as promised to work on this.

Reply:
> No, I did not forget to remove this, the test is actually dependent which isn't supposed to be. Hence it fails when it should not. @timi-codes as promised to work on this.

@tolumide-ng Don't you think it will be nice you discuss with @timi-codes on this rather than skipping it?

Reply:
Thank you @kechyy , I have discussed with @timi-codes about this and he has promised to make a fix for the test that wouldn't be dependent in his next PR